inference-engine/src/gna_plugin/gna_plugin.cpp

```diff
@@ -711,6 +712,7 @@ void GNAPlugin::LoadNetwork(CNNNetwork & _network) {
         manager.register_pass<ngraph::pass::ConvertOpSet3ToOpSet2>();
         manager.register_pass<ngraph::pass::ConvertOpSet2ToOpSet1>();
         manager.register_pass<ngraph::pass::ConvertOpSet1ToLegacy>();
+        manager.register_pass<ngraph::pass::LSTMCellDecomposition>();
```

Comment:
Does LSTMCellDecomposition produce graph in legacy opset? Shouldn't we run it before transformations which covert graph from the latest to the legacy opset?